source/index.html.md

```diff
@@ -630,6 +630,31 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects. Currenlty we only
+accept the JWE.
+Your system must extract the content encryption key (<code>cek</code>) used to encrypt the JWE
+payload and Base64-encode it, along with the entire JWE.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwe]</dt>
+  <dd>
+    Type: <a href="https://tools.ietf.org/html/rfc7516#section-3.1">JWE Compact serialization</a>
+    <br>
+    <i>Required</i>
+    <br>
+    Complete payment token as received from Samsung Pay.
+  </dd>
+  <dt>samsungpay[symmetric_key]</dt>
```

Comment:
```suggestion
  <dt>samsungpay[cek]</dt>
```
... permaybehaps? Then the description below could introduce the "CEK" acronym (and we could shorten the introductory text)?

Reply:
It could be changed as you suggested, but I'd rather not remove the reference from the introduction text. I want to be clear about what has to happen. Users should not just encounter `cek` and then have to look up where that comes from. I want to be explicit.

Reply:
I think it is obvious that we do not disagree that it should be clear what has to happen. Though, I suspect we may disagree that the only way to achieve this is to introduce "CEK" in the text before the parameter 🙂

I think it's not much different than encountering `currency` and then in the next line notice that it's the alpha-3 version 🙂

However, I am certainly [not sure about the naming](https://twitter.com/secretGeek/status/7269997868) — if `symmetric_key`, or `cek` or something else is the better name.

Reply:
I've changed it around a bit, forgot to push the changes :)

Please give it a look now.